src/feature-lifecycle/tracking-issues.md

```diff
@@ -1,16 +0,0 @@
-# Using tracking issues
-
-**Status:** Stub
-
-Tracking issues are used to facilitate discussion and report on the status of standard library features. All public APIs need a dedicated tracking issue. Some larger internal units of work may also use them.
-
-## Creating a tracking issue
-
-There's a template that can be used to fill out the initial tracking issue. The Libs team also maintains [a Cargo tool](https://github.com/rust-lang/libs-team/tree/main/tools/unstable-api) that can be used to quickly dump the public API of an unstable feature.
```

Comment:
can you please link the `unstable-api` tool from the libs tracking issue template? https://github.com/rust-lang/rust/issues/new?template=library_tracking_issue.md

Reply:
I'm pretty sure the tool is broken. It hasn't worked for a long time now. That's why I removed it.

Reply:
😞 fair enough